lib/class-wp-theme-json-gutenberg.php

```diff
@@ -180,6 +181,18 @@ class WP_Theme_JSON_Gutenberg {
 				),
 			),
 		),
+		array(
+			'path'          => array( 'color', 'duotone' ),
+			'value_key'     => 'slug',
+			'css_var_infix' => 'duotone',
+			'value_wrapper' => 'url(#wp-duotone-filter-$s)',
```

Comment:
This is needed so that we can modify the output of `value`. The problem is how can we connect the CSS variable to the SVG filter in a way that doesn't require the user to understand what's going on under the hood.